setup.py

```diff
@@ -3,18 +3,28 @@
 #===================================================================================================
 # Imports
 #===================================================================================================
-from distutils.core import setup
+from setuptools import setup
 from vmpooler_client.version import version
 
+classifiers = [
+    'Development Status :: 5 - Production/Stable',
+    'Intended Audience :: End Users/Desktop',
+    'Topic :: Utilities',
+    'License :: OSI Approved :: Apache Software License',
+    'Programming Language :: Python :: 2',
+    'Programming Language :: Python :: 2.7',
+]
```

Comment:
I couldn't find what pep8 block assignment formatting is with a quick google, but the pep8 linter doesn't complain. Is that the hanging comma or the hanging bracket?